### PR TITLE
W frontend

### PR DESCRIPTION
**Pull request by Saidac**
Show view proto type done

### CODE REVIEW

File: migrations/20161207204908-create-checker.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      UserId: {
+        type: Sequelize.INTEGER
+      },
+      TaskId: {
+        type: Sequelize.INTEGER
```
Comment (izelnakri) [inline migrations/20161207204908-create-checker.js:15]: you need foreign key constraints here

File: migrations/20161207204908-create-checker.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  up: function(queryInterface, Sequelize) {
+    return queryInterface.createTable('Checkers', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      UserId: {
+        type: Sequelize.INTEGER
```
Comment (izelnakri) [inline migrations/20161207204908-create-checker.js:12]: you need foreign key constraints here

File: views/chainz/index.pug [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+            td(rowspan="4"): h1 €80
+          tr
+            td: h6 task name
+          tr
+            td: h5 Participant
+            td(rowspan="2" class="checker"): h1 x
+            td(rowspan="2" class="checker"): h1 x
+            td(rowspan="2" class="checker"): h1 x
+            td(rowspan="2" class="today")
+              form(action="/checkers" id="checkerForm" method="POST" style="display:inline!important;")
```
Comment (izelnakri) [inline views/chainz/index.pug]: move inline style to component definition + try not to use !important

**Comment by Saidac** [comment]
Zahra has foreign key constraint in her branch. Refactor scss component file after your comments.

**Pull request merged**